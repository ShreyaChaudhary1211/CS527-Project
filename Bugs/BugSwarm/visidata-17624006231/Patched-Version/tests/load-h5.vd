sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/sensors.h5	o	open file
sensors		0	dive-row		^J	open this group or dataset
sensors_hchoi-20160128pk		0	dive-row		^J	open this group or dataset
sensors_hchoi-20160128pk_bonjour-BP_L		0	dive-row		^J	open this group or dataset
