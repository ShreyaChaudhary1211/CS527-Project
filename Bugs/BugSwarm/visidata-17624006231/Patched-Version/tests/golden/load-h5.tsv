elapsed_ms	timestamp	duration	arm	posture	systolic	diastolic
18000	1454017862000	10000	0	0	120	80
30000	1454017874000	1000	1	0	121	80
